backend/gsr_booking/api_wrapper.py

```diff
@@ -358,14 +176,15 @@ def update_token(self):
             "client_secret": settings.LIBCAL_SECRET,
             "grant_type": "client_credentials",
         }
+        
         response = requests.post(f"{API_URL}/1.1/oauth/token", body).json()
 
         if "error" in response:
             raise APIError(f"LibCal: {response['error']}, {response.get('error_description')}")
         self.expiration = timezone.localtime() + datetime.timedelta(seconds=response["expires_in"])
         self.token = response["access_token"]
-
-    def request(self, *args, **kwargs):
+    
+    def request(self, *arg, **kwargs):
```

Comment:
Hm, seems like you did this multiple times, any reason why we name to `arg`?

Reply:
think copilot suggested